Makefile

```diff
@@ -52,7 +52,7 @@ main.txt: main.json $(pcf)
 	nextpnr-ice40 --asc main.txt --top main --json main.json --pcf $(pcf) --freq $(MHz) --quiet --log nextpnr.log $(nextpnr_target) $(nextpnr_flags)
 icetime.json: main.txt $(pcf)
 	@echo "***** Timing analysis..."
-	icetime main.txt -c $(MHz) -j icetime.json $(icetime_flags)
+	icetime -c $(MHz) -j icetime.json $(icetime_flags) main.txt
```

Comment:
The version of icetime I was using required the input file to be after the flags per `icetime [options] input.asc`. I'm not really certain which version this is or if this will break other toolchains